index.html

```diff
@@ -1232,7 +1232,183 @@ <h3>signatureAlgorithm</h3>
 }
         </pre>
       </section>
+
+      <section id="EcdsaSecp256k1Signature2019" about="https://w3id.org/security#EcdsaSecp256k1Signature2019"
```

Comment:
Might be better if these were sorted with the above items rather than all at the bottom.

Reply:
I think its probably better to keep them grouped at the bottom until they are defined... that way when we define them, we can move them to the right place as we go.